_rules/inline_link_in_paragraph_is_distinguishable_be4d0c.md

```diff
@@ -0,0 +1,252 @@
+---
+id: be4d0c
+name: Inline link is distinguishable
+rule_type: atomic
+
+description: |
+  This rule checks that links that are embedded in a paragraph, list or cell have a way to distinguish them as links compared to the rest of the text
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+
+acknowledgements:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+  previous authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any HTML element which:
+
+- has the [semantic role][] of ['link'][link] or a [semantic role][] that inherits from the ['link'][link] role; and
+- is a [descendant][] of a `p`, `li` or `td` HTML elements (henceforth referred to as the _ancestor_ element) in the [flat tree][]; and
```

Comment:
What's the reason for this scope? Why just these three elements, and not headings for instance, or a div with role=listitem? This should be either explained in a note, or changed.

Reply:
The rule looks at links within paragraphs. We agreed that besides the `p` element, text within list items and table cells would be considered paragraphs also. But a heading should not be a paragraph. 
I've extended the applicability to consider lisitem and cell roles